Tests/Brokerages/GDAX/GDAXTestHelpers.cs

```diff
@@ -9,6 +24,7 @@ namespace QuantConnect.Tests.Brokerages.GDAX
 {
     public class GDAXTestsHelpers
     {
+        private static readonly Symbol Btcusd = Symbol.Create("BTCUSD", SecurityType.Crypto, Market.GDAX);
```

Comment:
No need to change, but could alternatively use `Symbols.BTCUSD` from within the test project.